_studio/mfx_lib/shared/src/mfx_vc1_dec_common.cpp

```diff
@@ -117,14 +117,6 @@ mfxStatus MFXVC1DecCommon::Query(VideoCORE* core, mfxVideoParam *in, mfxVideoPar
                 break;
             }
 
-            if ((in->IOPattern & MFX_IOPATTERN_OUT_SYSTEM_MEMORY) || (in->IOPattern & MFX_IOPATTERN_OUT_VIDEO_MEMORY) ||
```

Comment:
well, it's obvious that previous code had logic issue, but I don't think that we should remove this piece completely. Most likely it should be done similarly with other codecs, see [here](https://github.com/Intel-Media-SDK/MediaSDK/blob/95e1735206bfd91cb5c6ae809ebb85c87f0db599/_studio/shared/umc/codec/h265_dec/src/umc_h265_mfx_utils.cpp#L777)

Reply:
Setting 'unsupported' status definitely makes sense.